octree/include/pcl/octree/octree_key.h

```diff
@@ -138,9 +138,14 @@ class OctreeKey {
                                       (!!(this->z & depthMask)));
   }
 
-  /* \brief maximum depth that can be addressed */
-  static const unsigned char maxDepth =
-      static_cast<unsigned char>(sizeof(uindex_t) * 8);
```

Comment:
Would it be possible to keep this and deprecate it? Same for `max_sample_checks_`.

Reply:
I guess so - not sure if it will do linking errors downstream though.